test/hotspot/jtreg/compiler/loopopts/Test8335708.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @bug 8335708
+ * @summary Crash Compile::verify_graph_edges
+ * @requires vm.debug == true & vm.flavor == "server"
+ * @library /test/lib
+ *
+ * @run main/othervm
+ *       -XX:-TieredCompilation -XX:+VerifyGraphEdges
+ *       -XX:+StressIGVN -Xcomp
+ *       -XX:CompileCommand=compileonly,compiler.loopopts.Test8335708::mainTest
+ *       compiler.loopopts.Test8335708
```

Comment:
Can you please add a run without any flags? Sometimes that allows other bugs to trigger, because it can then be used without any flags, or other flag combinations.

Reply:
Done. I hope it's the right way.